graveler/testutil/fakes.go

```diff
@@ -20,10 +20,17 @@ type CommittedFake struct {
 	diffIterator  graveler.DiffIterator
 	Err           error
 	TreeID        graveler.TreeID
-	TreeExists    bool
 	AppliedData   AppliedData
 }
 
+type Tree struct {
```

Comment:
```suggestion
type TreeFake struct {
```
otherwise `testutil.Tree` implements `graveler.Tree` and confuses me.